test/test-integration/src/test/resources/sofa-rpc/rpc-config.json

```diff
@@ -1,4 +1,6 @@
 {
   "rpc.config.order": 999,
-  "logger.impl": "com.alipay.sofa.rpc.log.SLF4JLoggerImpl"
+  "logger.impl": "com.alipay.sofa.rpc.log.SLF4JLoggerImpl",
+  "consumer.connect.elatic.size":2,
+  "consumer.connect.elatic.precent":0
```

Comment:
No need change in `test-integration/src/test/resources/sofa-rpc/rpc-config.json`